src/main/java/goodgenerator/blocks/tileEntity/ComponentAssemblyLine.java

```diff
@@ -30,15 +33,18 @@
 import gregtech.api.interfaces.ITexture;
 import gregtech.api.interfaces.metatileentity.IMetaTileEntity;
 import gregtech.api.interfaces.tileentity.IGregTechTileEntity;
+import gregtech.api.metatileentity.implementations.GT_MetaTileEntity_Hatch_InputBus;
 import gregtech.api.render.TextureFactory;
 import gregtech.api.util.GT_Multiblock_Tooltip_Builder;
 import gregtech.api.util.GT_Recipe;
 import gregtech.api.util.GT_StructureUtility;
+import gregtech.api.util.GT_Utility;
 
 public class ComponentAssemblyLine extends GT_MetaTileEntity_LongPowerUsageBase<ComponentAssemblyLine>
         implements ISurvivalConstructable {
 
     private int casingTier;
+    private boolean separateInputBuses = false;
```

Comment:
@Dream-Master if we want to include https://github.com/GTNewHorizons/GT5-Unofficial/pull/1767 for next stable, then we shouldn't declare input separation here. The mentioned PR introduces a flag for input separation in the GT5U base class, to use with the GUI buttons.

Reply:
fixed in [bf973d7](https://github.com/GTNewHorizons/GoodGenerator/pull/136/commits/bf973d7df0f3c8dbf3ef7da9e360fd84445691ad)